src/index.js

```diff
@@ -1,11 +1,11 @@
 const description = document.querySelector('.text');
```

Comment:
- [ ] Your project should have one new JavaScript files imported as a module, and it should contain methods related to the status updates